src/components/pages/data/state-data.js

```diff
@@ -24,20 +24,26 @@ const State = ({ state }) => (
       {state.twitter && (
         <li>
           <a href={`https://twitter.com/${state.twitter}`}>
-            {state.name} on Twitter
+            <span className="a11y-only">{state.name}&apos;s</span>
+            Official Twitter
           </a>
+          <span>{'\u2022'}</span>
         </li>
       )}
       {state.covid19Site && (
-        <li>
-          <a href={state.covid19Site}>
-            Best current data source for {state.name}
-          </a>
-        </li>
+        <>
+          <li>
+            <a href={state.covid19Site}>
+              <span className="a11y-only">{state.name}&apos;s</span>
+              Official Data Source
```

Comment:
Let's change this to "Best Current Data Source"

Reply:
Also, thanks so much for this, the pages are looking way shorter!

Reply:
No problem! Thanks for the feedback.